cloudformation.json

```diff
@@ -2,6 +2,21 @@
   "AWSTemplateFormatVersion" : "2010-09-09",
   "Description": "CloudWatch to Humio Integration for sending CloudWatch Logs and Metrics to Humio.",
   "Parameters" : {
+    "ConfigureIngestLambdaVpcConfig": {
+      "Type": "String",
+      "AllowedValues" : [
+        "true",
+        "false"
+      ]
+    },
+    "SecurityGroupIds" : {
+      "Type": "List<AWS::EC2::SecurityGroup::Id>",
```

Comment:
Hmm. I understand that it is very neat to use the AWS::EC2::SecurityGroup::Id as the type here and the AWS::EC2::Subnet:Id below, however, this makes these parameters a requirement for setting up the integration. If a user does not want to use the VPC option, then they would have to create security groups (there may actually be a default value for this one?) and subnets anyway to fill out these values as CloudFormation validates them. 

So, I am afraid that you will have to use the "CommaDelimitedList" as the "Type" property here instead, which would require that you manually write these values... Unless you have a better idea about how not to make these parameters require defined security groups / subnets? :thinking: